app/services/open_library/book_service.rb

```diff
@@ -0,0 +1,33 @@
+module OpenLibrary
+  class BookService < BaseService
+    def find_by(params)
+      params_string = params.map { |e| e.join(':') }.join(',')
+
+      url = '/books'
+
+      result = self.class.get(url, {
+                                query: {
+                                  format: @format,
+                                  bibkeys: params_string,
+                                  jscmd: @jscmd
+                                }
+                              }).body
+
+      JSON.parse(result)
+    end
+
+    def find(isbn)
```

Comment:
```suggestion
    def find_by_isbn(isbn)
```

A final de cuentas solo busca por ISBN

Reply:
en ese caso seria mejor renombrar find_by y lo dejo solo para isbn, ya que seria redundante con dos funciones que se pueden llamar similar:
```
find_by ISBN: '345345345'
find_by_isbn '345345345'
```